books/book.js

```diff
@@ -0,0 +1,42 @@
+let currentUrl = new URL(location.href); // creating new url that contains current url
+let searchParams = new URLSearchParams(currentUrl.search); //  searching in current  url params
+let currentUrlId = searchParams.get("id"); //  getting current url id
+window.addEventListener("DomContentLoaded", getBook());
+
+function getBook() {
+  fetch("https://bookshop-api.mirkwood.dev/books/")
```

Comment:
getBook should call the https://bookshop-api.mirkwood.dev/books/id endpoint to get a single book and not all the books. It is not necessary to get all the books just to display info about a single book. Imagine you have 1 million books and used this approach - the user would wait for a long time to get a massive amount of data even though they don't need to.